omas/omas_physics.py

```diff
@@ -593,7 +602,7 @@ def summary_lineaverage_density(ods, line_grid=2000, time_index=None, update=Tru
 
     if time_index is None:
         for time_index in range(len(ods['core_profiles']['profiles_1d'])):
-            summary_lineaverage_density(ods_n, line_grid=line_grid, time_index=time_index, update=False)
+            summary_lineaverage_density(ods_n, line_grid=line_grid, time_index=time_index, update=update, doPlot=doPlot)
```

Comment:
`update` should always be set to `false` here, since whether ods is updated is already set previously